src/App.js

```diff
@@ -22,33 +22,37 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1)
+      // 1) drawCard is a function utilizing the Math built in method with .floor and .random on both of the arrays contained within the suit and rank variables to create a new output by multiplying the entire length of the array. 
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
       suit[Math.floor(Math.random() * suit.length)]
     }`
-    // 2)
+      // 2) Conditional statement starts off with if hand.indexOf the vairable draw which is absolutely equated to -1 then return the setter function
```

Comment:
You are correct, but to go deeper, the reason for being evaluated to -1 is to see if the current card already exists in the hand array.